src/PhysicalModels/MechanicalModels.jl

```diff
@@ -95,67 +95,110 @@ struct EnergyInterpolationScheme{M1<:Mechano,M2<:Mechano} <: PhysicalModel
 end
 
 
-struct ComposedElasticModel <: Elasto
-  Model1::Elasto
-  Model2::Elasto
+struct MultiElastic{A,B} <: Elasto
+  Model1::A
+  Model2::B
   Kinematic
-  function ComposedElasticModel(model1::Elasto,model2::Elasto)
-    @assert model1.Kinematic == model2.Kinematic
-    new(model1,model2,model1.Kinematic)
+  function MultiElastic(model1::A,model2::B) where{A,B}
+    @assert (model1 isa Elasto) == (model2 isa Elasto)
+    new{A,B}(model1,model2,model1.Kinematic)
+  end
+  function (obj::MultiElastic{<:AnisoElastic,<:AnisoElastic})(Λ::Float64=1.0)
+    DΨ1 = obj.Model1(Λ)
+    DΨ2 = obj.Model2(Λ)
+    Ψ, ∂Ψ, ∂∂Ψ = map((ψ1,ψ2) -> (x,N) -> ψ1(x,N) + ψ2(x,N), DΨ1, DΨ2)
+    return (Ψ, ∂Ψ, ∂∂Ψ)
+  end
+  function (obj::MultiElastic{<:IsoElastic,<:AnisoElastic})(Λ::Float64=1.0)
+    DΨ1 = obj.Model1(Λ)
+    DΨ2 = obj.Model2(Λ)
+    Ψ, ∂Ψ, ∂∂Ψ = map((ψ1,ψ2) -> (x,N) -> ψ1(x) + ψ2(x,N), DΨ1, DΨ2)
+    return (Ψ, ∂Ψ, ∂∂Ψ)
   end
-  function (obj::ComposedElasticModel)(Λ::Float64=1.0)
+  function (obj::MultiElastic{<:AnisoElastic,<:IsoElastic})(Λ::Float64=1.0)
+    DΨ1 = obj.Model1(Λ)
+    DΨ2 = obj.Model2(Λ)
+    Ψ, ∂Ψ, ∂∂Ψ = map((ψ1,ψ2) -> (x,N) -> ψ1(x,N) + ψ2(x), DΨ1, DΨ2)
+    return (Ψ, ∂Ψ, ∂∂Ψ)
+  end
+  function (obj::MultiElastic{<:IsoElastic,<:IsoElastic})(Λ::Float64=1.0)
```

Comment:
I'm very afraid of this: there are two possible ways to do the mase thing. Typically, it'll end with bringing headaches